.travis.yml

```diff
@@ -4,6 +4,5 @@ rvm:
   - 2.1.0
   - 2.0.0
 env:
-  - "AS_VERSION=3.2.16"
-  - "AS_VERSION=4.0.2"
+  - "AS_VERSION=4.2"
```

Comment:
This was causing issues with our minitest version, imo we don't need to support this old stuff.

Reply:
For sure. I think even 5.2 would be fine.